From f653c1b5cdcd4e3012829b7ff90c1e81f8572be3 Mon Sep 17 00:00:00 2001
From: Tim Niemueller <niemueller@kbsg.rwth-aachen.de>
Date: Tue, 31 May 2016 23:03:59 +0200
Subject: [PATCH] skiller: fix spurious re-execution on releasing control

When exclusive control was released, a previously executed skill could
be executed again as one internal variable was not reset.
---
 src/plugins/skiller/exec_thread.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/plugins/skiller/exec_thread.cpp b/src/plugins/skiller/exec_thread.cpp
index 93df070..dd49ef2 100644
--- a/src/plugins/skiller/exec_thread.cpp
+++ b/src/plugins/skiller/exec_thread.cpp
@@ -500,6 +500,7 @@ SkillerExecutionThread::loop()
 			  m->sender_thread_name());
 	
 	__continuous_reset = true;
+	curss = "";
 	__last_exclusive_controller = __skiller_if->exclusive_controller();
 	__skiller_if->set_exclusive_controller(0);
 	write_skiller_if = true;
-- 
2.5.5

